railties/Rakefile

```
@@ -124,6 +129,19 @@ namespace :test do
     end
 
     puts "--- All tests completed"
```

```suggestion
    puts "+++ All tests completed"
```

So it's easier to notice.

---

Now this block, with the profile always shows up expanded.
Previously this block would get replace if there were failing tests, hench the `^^^ +++`, that will now be a separate block which will be expanded.
I think that's desirable, just making note.